setup.py

```diff
@@ -4,7 +4,7 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-VERSION = os.environ.get("RAY_PROVIDER_VERSION", "0.2.0")
+VERSION = os.environ.get("RAY_PROVIDER_VERSION", "0.2.1")
```

Comment:
```suggestion
VERSION = "0.2.1"
```

Reply:
I strongly prefer to keep this as is --  this change is 1. non-consequential and 2. introduces a greater burden on the releaser when we do RCs